command/acl/token/update/token_update_test.go

```diff
@@ -109,6 +109,22 @@ func TestTokenUpdateCommand(t *testing.T) {
 		require.ElementsMatch(t, expected, token.NodeIdentities)
 	})
 
+	// update with append-node-identity
+	t.Run("append-node-identity", func(t *testing.T) {
+
+		token := run(t, []string{
+			"-http-addr=" + a.HTTPAddr(),
+			"-accessor-id=" + token.AccessorID,
+			"-token=root",
+			"-append-node-identity=third:node",
+			"-description=test token",
+		})
+
+		require.Len(t, token.NodeIdentities, 3)
+		require.Equal(t, "third", token.NodeIdentities[2].NodeName)
+		require.Equal(t, "node", token.NodeIdentities[2].Datacenter)
+	})
```

Comment:
FYI subtests using `t.Run` should be independently runnable and pass. These append tests will not pass unless the previous subtests pass (but the previous test is also dependent on the previous-previous one).

As a team convention we tend to use `testutil.RunStep` to indicate a sequence of tests that must be run in a specific order.

Not a blocker, but just something to keep in mind as you write tests.